### PR TITLE
exactInference test added for markov model

### PR DESCRIPTION
**Pull request by anaviltripathi**
#571 Separated the features.

### CODE REVIEW

File: pgmpy/tests/test_inference/test_ExactInference.py [hunk 1]
```diff
@@ -98,44 +98,133 @@ def test_query_multiple_times(self):
...
 
     def test_induced_width(self):
-        result_width = self.bayesian_inference.induced_width(['G', 'Q', 'A', 'J', 'L', 'R'])
+        result_width = self.markov_inference.induced_width(['G', 'Q', 'A', 'J', 'L', 'R'])
         self.assertEqual(2, result_width)
 
     def tearDown(self):
-        del self.bayesian_inference
-        del self.bayesian_model
+        del self.markov_inference
```
Comment (khalibartan) [inline pgmpy/tests/test_inference/test_ExactInference.py]: You shouldn't have removed the tear down for BayesianModels .

**Comment by anaviltripathi** [comment]
@ankurankan Did I miss some tests or is there some other problem in the tests that I have written?

**Comment by khalibartan** [comment]
@anaviltripathi ankur might be busy, that's why he hasn't given a look.

**Comment by ankurankan** [comment]
@anaviltripathi I did some minor changes before merging. You can see them in this commit: df7b6b564d770ae283e16588bcccfcf562002d83

**Pull request merged**